web/src/user_group_edit.ts

```diff
@@ -1584,6 +1674,10 @@ export function update_displayed_groups(filter_id: string): void {
         $(".user-groups-list").removeClass("hide-deactived-user-groups");
         $(".user-groups-list").removeClass("hide-active-user-groups");
     }
+    if (filters_dropdown_widget) {
+        filters_dropdown_widget.render(filter_id);
```

Comment:
Why is this needed here? For most cases `updata_displayed_groups` is just passed the value by calling `widget.value()` and then same value is being rendered again.